server/src/test-data.js

```diff
@@ -105,7 +105,16 @@ export const loadTestData = () =>
               name: 'VR Wollongong Escarpment',
               details: 'VR operators required to assist with rescue',
               group,
-            }),
+            }).then(event =>
+              Creators.eventResponse({
+                status: 'On Route',
```

Comment:
didn't you make this field an int?

Reply:
incorrectly yes, I have changed the model to be a string (for now anyway, maybe we do want ints?)

Reply:
yeah eventually but str is fine for now - easy to change later